mkdocs.yml

```
@@ -0,0 +1,69 @@
+site_name: Flux
+# site_description:
```

```suggestion
site_description: Documentation for Flux, the GitOps Kubernetes operator.
```

(Creates a description tag for SEO purposes on the index page).